components/Ang/utils/bakePankti.tsx

```diff
@@ -2,12 +2,19 @@ import React from 'react';
 import {View, Text} from 'react-native';
 import {getStyle} from './getStyle';
 
-export const bake = (verse: string, larivaarAssist: Boolean) => {
+export const bakePankti = (
+  verse: string,
+  larivaar: Boolean,
+  larivaarAssist: Boolean,
+) => {
```

Comment:
Recommended to have these arguments in object. Here's the benefit:

1. We don't have to firmly abide by the order of arguments while passing in function call
2. Extending the number of arguments without worrying about existing code is easy. Reduces the possibility of breaking change in the future.

Reply:
Done veerji